src/main/scala/org/dia/algorithms/mcc/NetcdfDFSMCC.scala

```diff
@@ -39,8 +39,8 @@ object NetcdfDFSMCC {
    * @return
    */
   def createMapFromEdgeList(edgesRDD: Iterable[MCCEdge],
-                            lat: Array[Double], lon: Array[Double]): mutable.HashMap[String, Any] = {
-    val MCCNodeMap = new mutable.HashMap[String, Any]()
+                            lat: Array[Double], lon: Array[Double]): mutable.HashMap[String, MCCNode] = {
+    val MCCNodeMap = new mutable.HashMap[String, MCCNode]()
     edgesRDD.foreach { edge => {
       val srcKey = s"${edge.srcNode.frameNum},${edge.srcNode.cloudElemNum}"
```

Comment:
Can we have the key formed as s"$frameNum:$cloudElemNum", this would help keep a single format for the string keys generated from the nodes. 

Or another way to avoid this confusion would be to have a method in the MCCNode object called getStringKey() that would return the above key. And hence we should use that everywhere else. I believe this would reduce confusion and future effort. Thoughts @kwhitehall ?